### ISSUE TITLE
Urine anion gap positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10639]
**Preferred term label:**
Urine anion gap positive

**Synonyms**
Na+ + K+ - Cl-

**Definition (free text, please give PubMed ID)**
https://www.mdcalc.com/calc/2045/urine-anion-gap

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal renal tubular acidosis 
https://www.ncbi.nlm.nih.gov/books/NBK547595/
"Absence of a negative urine anion gap (UAG) in an individual with metabolic acidosis. Calculation of the UAG (UAG=[Na+]U+[K+]U- [Cl-]U) can help to distinguish between primary forms of proximal and dRTA."

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
Positive urine anion gap
HP:0025763

The urine anion gap (UAG), calculated as [(Na+ + K+) - (Cl-)] has been used to roughly estimate whether urine ammonium is increased or decreased in the evaluation of hyperchloremic metabolic acidosis. The more ammonium in the urine, the lower the UAG is, which achieves negative values within a wide range (0 to -200 mEq/L) during hyperchloremic metabolic acidosis. In patients with distal renal tubular acidosis (RTA) of several etiologies, the UAG is positive, despite the prevailing metabolic acidosis, and this is a useful bedside diagnostic index of impaired ammonium excretion.
